### PR TITLE
Install required header files for building ntlm_auth_wrapper.

### PR DESCRIPTION
**Pull request by stgmsa**
# Description
Add required header files for building ntlm_auth_wrapper while setting up dev environment.

# Impacts
Install epel release, libcurl-devel, cjson-devel while setting up dev machine

# Delete branch after merge
YES

# Checklist
- [n/a] Document the feature
- [n/a] Add OpenAPI specification
- [n/a] Add unit tests
- [n/a] Add acceptance tests (TestLink)

## New Features
* install epel release while setting up dev environment
* install libcurl-devel and cjson-devel while setting up dev environment

# UPGRADE file entries
addons/dev-helpers/setup-dev-env.sh

### CODE REVIEW

File: addons/dev-helpers/setup-dev-env.sh [hunk 1]
```diff
@@ -31,6 +31,10 @@ git clone https://github.com/inverse-inc/packetfence /usr/local/pf
 log_section "Set the safe.directory in git"
 git config --global --add safe.directory /usr/local/pf
 
+log_section "install epel and required header files"
+dnf install -y epel-release
```
Comment (JeGoi) [inline addons/dev-helpers/setup-dev-env.sh]:
```suggestion
```
We do not need it.
libcurl and cjson are on our repo
And we will have issues after if this repo is activated by default.

**Review by JeGoi — CHANGES_REQUESTED** [review]

File: addons/dev-helpers/setup-dev-env.sh [hunk 2]
```diff
@@ -31,6 +31,10 @@ git clone https://github.com/inverse-inc/packetfence /usr/local/pf
...
+dnf install -y libcurl-devel cjson-devel
```
Comment (JeGoi) [inline addons/dev-helpers/setup-dev-env.sh]:
```suggestion
dnf install -y --enablerepo=packetfence libcurl-devel cjson-devel
```

File: addons/dev-helpers/setup-dev-env.sh (hunk 1 above)
Comment (stgmsa) [reply addons/dev-helpers/setup-dev-env.sh]: let me test it without installing epel release.

**Review by JeGoi — CHANGES_REQUESTED** [review]
Related to the previous review.
Already tested

File: addons/dev-helpers/setup-dev-env.sh [hunk 3]
```diff
@@ -31,6 +31,9 @@ git clone https://github.com/inverse-inc/packetfence /usr/local/pf
 log_section "Set the safe.directory in git"
 git config --global --add safe.directory /usr/local/pf
 
+log_section "install epel and required header files"
```
Comment (JeGoi) [inline addons/dev-helpers/setup-dev-env.sh]:
```suggestion
log_section "install required header files from PF repo"
```

**Review by JeGoi — CHANGES_REQUESTED** [review]
Fix title

**Review by stegar123 — APPROVED** [review]

**Pull request merged**